### PR TITLE
Examples: Convert LeePerrySmith.json to glTF.

### PR DESCRIPTION
**Pull request by donmccurdy**
Per #14737.

Demos:

- [decals](https://rawgit.com/donmccurdy/three.js/examples-leeperrysmith-glb/examples/webgl_decals.html)
- [helpers](https://rawgit.com/donmccurdy/three.js/examples-leeperrysmith-glb/examples/webgl_helpers.html)
- [loader_ctm](https://rawgit.com/donmccurdy/three.js/examples-leeperrysmith-glb/examples/webgl_loader_ctm.html)
- [materials_bumpmap](https://rawgit.com/donmccurdy/three.js/examples-leeperrysmith-glb/examples/webgl_materials_bumpmap.html)
- [materials_bumpmap_skin](https://rawgit.com/donmccurdy/three.js/examples-leeperrysmith-glb/examples/webgl_materials_bumpmap_skin.html)
- [materials_modified](https://rawgit.com/donmccurdy/three.js/examples-leeperrysmith-glb/examples/webgl_materials_modified.html)
- [materials_normalmap](https://rawgit.com/donmccurdy/three.js/examples-leeperrysmith-glb/examples/webgl_materials_normalmap.html)
- [materials_skin](https://rawgit.com/donmccurdy/three.js/examples-leeperrysmith-glb/examples/webgl_materials_skin.html)
- [modifier_simplifier](https://rawgit.com/donmccurdy/three.js/examples-leeperrysmith-glb/examples/webgl_modifier_simplifier.html)
- [postprocessing_advanced](https://rawgit.com/donmccurdy/three.js/examples-leeperrysmith-glb/examples/webgl_postprocessing_advanced.html)

### CODE REVIEW

File: examples/webgl_helpers.html [hunk 1]
```diff
@@ -67,18 +69,18 @@
 					// To make sure that the matrixWorld is up to date for the boxhelpers
 					group.updateMatrixWorld(true);
 
-					var mesh = new THREE.Mesh( geometry, material );
 					group.add( mesh );
 
-					fnh = new THREE.FaceNormalsHelper( mesh, 5 );
-					scene.add( fnh );
+					// TODO: Implement BufferGeometry support in FaceNormalsHelper.
```
Comment (Mugen87) [inline examples/webgl_helpers.html]: I'm not sure it makes sense if `FaceNormalsHelper` supports `BufferGeometry`. The helper would need to calculate the face normals before visualization. I think it's better if helpers just visualize data which are actually present in the underlying object.

Comment (WestLangley) [reply examples/webgl_helpers.html]: Right. `FaceNormalsHelper` does not apply to `BufferGeometry`.

Comment (donmccurdy) [reply examples/webgl_helpers.html]: OK, so just remove this bit from the example entirely?

Comment (donmccurdy) [reply examples/webgl_helpers.html]: Removed 👍

**Review by Mugen87 — APPROVED** [review]

**Comment by mrdoob** [comment]
Thanks!

**Pull request merged**